packages/ui/src/components/ListV2/HeaderRow.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from '@emotion/styled'
+import type { ReactNode } from 'react'
+import { Checkbox } from '../Checkbox'
+import { HeaderCell } from './HeaderCell'
+import { useListContext } from './ListContext'
+
+const StyledHeaderRow = styled.div`
+  /* List itself also apply style about common templating between HeaderRow and other Rows */
+  column-gap: ${({ theme }) => theme.space['2']};
+  padding: 0 ${({ theme }) => theme.space['2']};
+`
+
+const StyledCheckbox = styled(Checkbox)`
+  &[aria-disabled='true'] {
+    pointer-events: none;
+  }
+`
```

Comment:
Please avoid pointer event none, if we need to put a tooltip for some reason it won't work. What was the purpose of this?

Reply:
was similar in the current ListV2 (ListheaderRow), i change